docs/src/rules/no-implicit-globals.md

```diff
@@ -58,7 +58,7 @@ function bar() {}
 
 Examples of **correct** code for this rule:
 
-::: correct
+::: correct { "parserOptions": { "sourceType": "script" } }
```

Comment:
I'll think all the options we need in rules examples will be in `parserOptions`, so we could simplify this:

```
::: correct { "sourceType": "script" } 
```

And then translate this into config options in the container's render function. That would be also easier to update when we switch playground to the flat config.

Reply:
Love it! 911656e